contributed_definitions/nyaml/NXraman.yaml

```
@@ -0,0 +1,755 @@
+# - Data: measured data, data errors
+# - Derived parameters: e.g. extra parameters derived in the measurement software
+type: group
+NXraman(NXopt):
+  (NXentry):
+    doc: |
+      This is the application definition describing Raman spectroscopy experiments.
+      
+      Such experiments may be as simple a single Raman spectrum from spontanous
+      Raman scattering and range to Raman imaging Raman spectrometer,
+      surface- and tip-enhanced Raman techniques, x-Ray Raman scattering, as well
+      as resonant Raman scattering phenomena or multidimenional Raman spectra (i.e.
+      varying temperature, pressure, electric field, ....)
+      
+      The application definition defines:
+      
+      * elements of the experimental instrument
+      * calibration information if available
+      * parameters used to tune the state of the sample
+      * sample description
+    definition:
+      doc: |
+        An application definition for ellipsometry.
+      \@version:
+        doc: |
+          Version number to identify which definition of this application
+          definition was used for this entry/data.
+      \@url:
+        doc: |
+          URL where to find further material (documentation, examples) relevant
+          to the application definition.
+      enumeration: [NXraman]
+    experiment_description:
+      doc: |
+        An optional free-text description of the experiment.
+        
+        However, details of the experiment should be defined in the specific
+        fields of this application definition rather than in this experiment
+        description.
+    experiment_type:
+      doc: |
+        Specify the type of Raman measurement.
+      enumeration: [in situ Raman spectroscopy, resonant Raman spectroscopy, non-resonant Raman spectroscopy, Raman imaging, Tip-enhanced Raman spectroscopy (TERS), Surface-enhanced Raman spectroscopy (SERS), Surface plasmon polariton enhanced Raman scattering (SPPERS), Hyper Raman spectroscopy (HRS), Stimulated Raman spectroscopy (SRS), Inverse Raman spectroscopy (IRS), Coherent anti-Stokes Raman spectroscopy (CARS)]
+    (NXinstrument):
+      doc: |
+        Properties of the Instruments used for Raman instrumeasurements.
+      company:
+        exists: optional
+        doc: |
+          Name of the company which build the instrument.
+      construction_year(NX_DATE_TIME):
+        exists: optional
+        doc: |
+          ISO8601 date when the instrument was constructed.
+          UTC offset should be specified.
+      software(NXprocess):
+        program:
+          doc: |
+            Commercial or otherwise defined given name of the program that was
+            used to generate the result file(s) with measured data and metadata.
+            This program converts the measured signals to Raman data. If
+            home written, one can provide the actual steps in the NOTE subfield
+            here.
+      incident_source_wavelength(NX_NUMBER):
```

```suggestion
      incident_wavelength(NX_NUMBER):
```

What is the incident _source_? What other incident wavelengths are there?

---

Why is this not part of the NXsource or NXbeam group?